server/src/services/findStatAndRelateToTable.js

```diff
@@ -0,0 +1,8 @@
+import { Stat } from "../models/index.js"
+
+const findStatAndRelateToTable = async (stat, table) => {
+  let currentStat = await Stat.query().where({value: stat})
+  await table.$relatedQuery("stats").relate(currentStat)
+}
```

Comment:
i love that you moved this logic out, but we actually would prefer to define this method as an instance or static method on the Stat model. You could define it as an instance method if you grabby the Stat object with .where before running the method, otherwise you could make a static method like below

```
class Stat {
   static relateToTable(stat, table){
       ///your code 
    }
}
```

Reply:
```
await Stat.query().where({value: stat})

class Stat {
      relateToTable(table){
          await this.constructor.$relatedQuery("tables").relate(table)
    }
}
```

Im blanking on whether or not you need .constructor for this one, but you might